scripts/git-hooks/check_cli_doc.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
Finally, another developer that likes shell as much as I do?

Since this is the first hook, would be good to engage team in preferred implmentation language. @cyrush may have a rationale for using a different implementation language though, like Python.

Reply:
I'm actually very new to shell, so jury is still out on how much I like it, but I'm open to it. My first thought was to do this in python, but that requires supporting a git-python module that I did not want to force as a requirement on our repo in order to make commits.

Reply:
That makes sense to me.